project/Sheng08/README.md

```diff
@@ -0,0 +1,253 @@
+# OrientMesh Toolkit: Extracting and Visualizing Oriented Bounding Boxes from 3D Triangular Meshes
+
+## Basic Information
+
+> **Project Overview:** Extracting and Visualizing Oriented Bounding Boxes (OBB) from Triangular Meshes using C++ and Python
+
+Triangular meshes[^1] stand as a cornerstone in computer graphics, essential in the portrayal and animation of 3D objects. A critical aspect of processing these meshes involves extracting **Oriented Bounding Boxes (OBB)**[^2]. _++OrientMesh Toolkit++_ presents an optimal solution to this task, fusing efficient algorithms with an interactive visual interface. This combination not only simplifies OBB extraction but also enhances user comprehension through vivid visual representations.
+
+- 📂 **GitHub Repository**: [https://github.com/Sheng08/OrientMesh-Toolkit](https://github.com/Sheng08/OrientMesh-Toolkit)
+- 💻 **Implemented In**:
+  - **Core Library**: C++
+  - **Bindings & Interface(GUI)**: Python
+
+## Problem to Solve
+
+Triangular meshes, due to their intricate detailing, can sometimes be complex and computationally heavy for operations. Having a precise oriented bounding box aids in simplifying many computational geometry problems, like collision detection, mesh simplification, and many more. However, generating an optimal OBB is a non-trivial task. OrientMesh aims to offer a reliable, optimized, and easily-integrable solution for this challenge.
```

Comment:
Is it surface mesh or body mesh?

Reply:
My project primarily focuses on the surface of the mesh to address the applications of computational geometry problems associated with surface meshes, like collision detection.
I've also revised the 'Problem to Solve' section to provide a clearer explanation. Thanks!

Reply:
Good